CMakeLists.txt

```diff
@@ -240,24 +240,26 @@ if(USE_MKLDNN)
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /EHsc /Gy /MT")
   endif()
 
-  set(MKLDNN_BUILD_TESTS OFF CACHE INTERNAL "" FORCE)
-  set(MKLDNN_BUILD_EXAMPLES OFF CACHE INTERNAL "" FORCE)
-  set(MKLDNN_ARCH_OPT_FLAGS "" CACHE INTERNAL "" FORCE)
-  set(MKLDNN_ENABLE_JIT_PROFILING OFF CACHE INTERNAL "" FORCE)
-  set(MKLDNN_LIBRARY_TYPE STATIC CACHE INTERNAL "" FORCE)
-  set(DNNL_ENABLE_CONCURRENT_EXEC ON CACHE INTERNAL "" FORCE)
-
-  if(NOT USE_OPENMP)
+  function(load_mkldnn)
+    set(MKLDNN_BUILD_TESTS OFF CACHE INTERNAL "" FORCE)
+    set(MKLDNN_BUILD_EXAMPLES OFF CACHE INTERNAL "" FORCE)
+    set(MKLDNN_ARCH_OPT_FLAGS "" CACHE INTERNAL "" FORCE)
+    set(MKLDNN_ENABLE_JIT_PROFILING OFF CACHE INTERNAL "" FORCE)
+    set(MKLDNN_LIBRARY_TYPE STATIC CACHE INTERNAL "" FORCE)
+    set(DNNL_ENABLE_CONCURRENT_EXEC ON CACHE INTERNAL "" FORCE)
+
+    if(NOT USE_OPENMP)
       set(MKLDNN_CPU_RUNTIME SEQ CACHE INTERNAL "" FORCE)
-  endif()
-
-  add_subdirectory(3rdparty/mkldnn)
+    endif()
 
+    set(CMAKE_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}/mkldnn")
+    add_subdirectory(3rdparty/mkldnn)
+  endfunction()
+  load_mkldnn()
   include_directories(3rdparty/mkldnn/include)
   include_directories(${PROJECT_BINARY_DIR}/3rdparty/mkldnn/include)
   add_definitions(-DMXNET_USE_MKLDNN=1)
   list(APPEND mxnet_LINKER_LIBS dnnl)
-  set(INSTALL_MKLDNN ON)
```

Comment:
Why it's removed? @yzhliu FYI.

Reply:
This variable isn't used anywhere but in this line